controllers/administration.js

```diff
@@ -428,10 +428,25 @@ const getCSVImportHandler = () => {
                 },
             });
             const numberOfUsers = (stats.users.successful || 0) + (stats.users.failed || 0);
```

Comment:
does this line work if no users where importet or do we need another fallback like `((stats.users || {}).súccessfull || 0) ...` ?

Reply:
To be honest, I think I'll reduce it even further to `stats.users.successful + stats.users.failed`, as this is the default value with wich the statistics are initialized server-side.